java-frontend/src/main/java/org/sonar/java/se/checks/NullDereferenceCheck.java

```diff
@@ -56,7 +56,7 @@
 public class NullDereferenceCheck extends SECheck {
 
   private static final ExceptionalYieldChecker EXCEPTIONAL_YIELD_CHECKER = new ExceptionalYieldChecker(
-    "NullPointerException will be thrown when invoking method %s().");
+    "\"NullPointerException\" will be thrown when invoking method \"%s()\".");
```

Comment:
I updated the messages to be coherent with the "normal" null dereference issue message. 
I also wanted to add a dot at the end of the normal message, but it was too many updates not related to the fix.

If it's OK for you I'm going to update the messages for other rules using exception on method invocation messages, and add a second commit to this PR with the update, in order to not pollute the fix.